bin/container.sh

```diff
@@ -175,39 +166,24 @@ start_cassandra() {
 }
 
 console_cassandra() {
-  wcip=$(docker inspect --format "{{ .NetworkSettings.Networks.${docker_network}.IPAddress }}" ${project}-cassandra)
-
-  docker run --net=${docker_network} -it --rm ${cassandra} cqlsh ${wcip} || \
-    usage "unable to run command: docker run --net=${docker_network} -it --rm ${cassandra} cqlsh ${wcip}" 1
+  docker exec -it ${project}-cassandra cqlsh || \
```

Comment:
I see, makes sense. On windows I just used `exec` because it just worked and wondered why we would introduce the additional complexity. I will try to simplify `run` then so that we don't need the `${wcip}` but can use `${project}-cassandra` instead as well.